frontend/src/pages/clusterSettings/ModelServingPlatformSettings.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { Alert, AlertActionCloseButton, Checkbox, Stack, StackItem } from '@patternfly/react-core';
+import SettingSection from '~/components/SettingSection';
+import { ModelServingPlatformEnabled } from '~/types';
+
+type ModelServingPlatformSettingsProps = {
+  initialValue: ModelServingPlatformEnabled;
+  enabledPlatforms: ModelServingPlatformEnabled;
+  setEnabledPlatforms: (platforms: ModelServingPlatformEnabled) => void;
+};
+
+const ModelServingPlatformSettings: React.FC<ModelServingPlatformSettingsProps> = ({
+  initialValue,
+  enabledPlatforms,
+  setEnabledPlatforms,
+}) => {
+  const [alert, setAlert] = React.useState<string>();
+
+  React.useEffect(() => {
+    setEnabledPlatforms(initialValue);
+  }, [initialValue, setEnabledPlatforms]);
+
+  React.useEffect(() => {
+    if (!enabledPlatforms.kServe && !enabledPlatforms.modelMesh) {
+      setAlert(
+        'Disabling both model serving platforms prevents new projects from deploying models. Models can still be deployed from existing projects that already have a serving platform.',
```

Comment:
@vconzola as I mentioned in the other comment, allowing disabling both, would create another edge case, so we need to check what's the best UI for that scenario (user cannot add new model servers)